python/pyarrow/_dataset.pyx

```diff
@@ -2393,53 +2434,6 @@ cdef class Scanner(_Weakrefable):
 
     A scanner is the class that glues the scan tasks, data fragments and data
     sources together.
-
-    Parameters
```

Comment:
I removed these since the default constructor only accepts a pointer to another dataset, and doesn't accept these arguments.